Wack A Devil.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
+<title>Wack A Devil</title>
+<head>
+  <title>Wack</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+</head>
+<style>
+  body {
+    background-image: url("devil.jpg");
+    text-align: center;
+
+  }
+
+  #score {
+    color: white;
+    margin: 20px;
+    font-size: 50px;
+  }
+
+  #game-board {
+    top: 30vh;
+    width: 190px;
+    height: 190px;
+    margin: auto;
+    border-top: 1px solid #111;
+    border-right: 1px solid #555;
+    border-bottom: 1px solid #555;
+    border-left: 1px solid #111;
+    background-color: #333;
+    position: relative;
+    cursor: pointer;
+  }
+
+  .devil {

```

Comment:
positioning - avoid manual calculation
instead, use 2 nested elements - parent and child
parent will have `position: relative` - so it will be the "anchor" for child
child will have `position: absolute`, which will be relative to parent